core/chains/evm/txmgr/eth_resender.go

```diff
@@ -112,11 +117,25 @@ func (er *EthResender) resendUnconfirmed() error {
 		if err != nil {
 			return errors.Wrap(err, "failed to FindEthTxAttemptsRequiringResend")
 		}
+		oldestAttempt, exists := findOldestUnconfirmedAttempt(attempts)
+		if exists {
+			// Wait at least 2 times the EthTxResendAfterThreshold to log critical with an unconfirmedTxAlertDelay
+			if (time.Since(oldestAttempt.CreatedAt) > er.config.EthTxResendAfterThreshold()*2) &&
+				time.Since(er.lastAlertTimestamps[k]) >= unconfirmedTxAlertDelay {
+				er.lastAlertTimestamps[k] = time.Now()
+				er.logger.Criticalw("TxAttempt has been uncofirmed for more than: ", er.config.EthTxResendAfterThreshold()*2,
+					"txID", oldestAttempt.EthTxID, "GasPrice", oldestAttempt.GasPrice, "GasTipCap", oldestAttempt.GasTipCap, "GasFeeCap", oldestAttempt.GasFeeCap,
```

Comment:
Could you also log the fromAddress here?
It will tell, which sending key is getting stuck.